src/tsdb.h

```diff
@@ -93,4 +94,26 @@ timestamp_t CalcWindowStart(timestamp_t timestamp, size_t window);
 timestamp_t getFirstValidTimestamp(Series *series, long long *skipped);
 
 CompactionRule *NewRule(RedisModuleString *destKey, int aggType, uint64_t timeBucket);
+
+typedef enum {
+    DICT_OP_SET = 0,
+    DICT_OP_REPLACE = 1,
+    DICT_OP_DEL = 2
+} DictOp;
+
+static inline int dictOperator(RedisModuleDict *d, void *chunk, timestamp_t ts, DictOp op) {
```

Comment:
Typically, static functions are functions needed in only one file, meaning they are declared static to make that explicit by limiting their visibility. 

If the function is indeed used in multiple files with identical definitions it should be made external, with a single definition. If only one *.c file actually uses it, the declaration does not belong in a header.

Reply:
static in header file many times resolves multi definition issues ([example](https://stackoverflow.com/questions/47819719/static-inline-functions-in-a-header-file)).

Reply:
change to only expose declaration in header...